templates/include/telemetry/environment.1.schema.json

```
@@ -890,6 +890,12 @@
                         "null"
                       ]
                     },
+                    "blocklisted": {
+                      "type": [
```

Since we're here, we may as well add a description field for this for future reference in the BigQuery console and the (upcoming) [schema dictionary](https://mozilla-schema-dictionary.netlify.app/). You can probably think of a better string than me:

```suggestion
                    "blocklisted": {
                      "description": "Indicates whether the driver was blocklisted (replaces blacklisted, see bug 1652842)",
                      "type": [
```

We should also add something similar for the previous `blacklisted` string.